package.json

```diff
@@ -7,6 +7,7 @@
   "author": "Anton Medvedev <anton@medv.io>",
   "homepage": "https://medv.io/codejar/",
   "main": "codejar.js",
+  "types": "codejar.d.ts",
```

Comment:
Does `.d.ts` need to be included in `"files"` below? To be in the package, or types added anyway?

Reply:
It's already covered by the "*.ts" entry. I tested it out by publishing to a local npm registry (Verdaccio) and it worked as expected.